remediation/code-samples/02-countValue.js

```diff
@@ -25,6 +27,7 @@ function countValue(array, value) {
   return array.length;
 }
 
+// PLEASE REVIEW
 function check(output, expected)
 {
   if (output.length !== expected.length){
```

Comment:
Similar to what was noted in 01-countEvens.js, be careful about what type of data you are working with. The function `countValue()` returns an Integer. We compare it against an Integer. Neither of these have a `length` property we can access.

Try modifying to compare the **values** of `output` and `expected`.